packages/gatsby-plugin-manifest/src/__tests__/gatsby-node.js

```diff
@@ -0,0 +1,48 @@
+const fs = require(`fs`)
+const { onPostBootstrap } = require(`../gatsby-node`)
+
+jest.mock(`fs`)
```

Comment:
These are hoisted to the top anyways, but would you mind adding it above line 1 (_on_ line 1) for clarity?

Reply:
Also looks like you assign things below as jest spies. You can also do this

```js
jest.mock(`fs`, () => {
  return {
    existsSync: jest.fn().mockImplementation(() => true),
    writeFileSync: jest.fn()
  };
});
```

Reply:
hm i thought imports are first then mocks. i removed the skipped line if that was the issue although i am not sure i understood your comment correctly. I am brought the mocks up as well but didnt do as you suggested since then i need to mock `statSync` but only for second test. i hope this is what you are looking for :)

Reply:
Nope! It's more for ES module stuff, but it does [hoist them to the top](https://jestjs.io/docs/en/manual-mocks#using-with-es-module-imports)--I think it actually uses [this babel plugin](https://www.npmjs.com/package/babel-plugin-jest-hoist)--so I prefer to just leave them at the top! Plus, at least to me, it clarifies things. I see a `jest.mock('fs-extra')` and then `require('fs-extra')` and I _know_ it's mocked if that makes sense!